src/main/frontend/components/sidebar.cljs

```diff
@@ -70,7 +70,7 @@
     (< delta 14)))
 
 (rum/defc page-name
-  [name icon]
+  [name icon display-name]
   (let [original-name (db-model/get-page-original-name name)]
     [:a {:on-click (fn [e]
                      (let [name (util/safe-page-name-sanity-lc name)]
```

Comment:
I guess we can handle the source name of the alias page in `on-click` event here, instead of the `favorites` component, so that this is no need to introduce the new `display-name` parameter.